awx/main/utils/licensing.py

```
@@ -551,3 +559,236 @@ def get_licenser(*args, **kwargs):
+    Returns (cert_pem, key_pem, consumer_uuid), any of which may be None if the
+    corresponding row is absent or on any DB error.  Best-effort: failures are
+    logged as warnings and never propagate.
+    """
+    all_keys = [CANDLEPIN_CERT_SETTING_KEY, CANDLEPIN_KEY_SETTING_KEY, CANDLEPIN_UUID_SETTING_KEY]
+    try:
+        from django.db import connection
+
+        placeholders = ', '.join(['%s'] * len(all_keys))
+        with connection.cursor() as cursor:
+            cursor.execute(
+                f'SELECT key, value FROM conf_setting WHERE key IN ({placeholders})',
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)